src/components/forms/file-dropper/file-dropper.tsx

```diff
@@ -126,6 +126,8 @@ const Description = styled.p`
   ${TextStyles.timestamp};
   letter-spacing: 0;
   margin: 0;
+  padding: 0 24px;
+  text-align: center;
```

Comment:
While making the screenshots, I noticed that padding and text align center are missing here on mobile

<img width="930" alt="Bildschirm­foto 2023-03-21 um 15 06 20" src="https://user-images.githubusercontent.com/57712895/226631559-24ea841c-40df-475a-937d-2f8f9a7567fd.png">